announcements/1.17.0.md

```diff
@@ -0,0 +1,78 @@
+
+### Start and stop support for containers
+
+Containers in Anbox Cloud have been ephemeral since the beginning and explicit start or stop of a container was not possible. This has now changed and an explicit `amc stop` or `amc start` command can be used to stop or start containers. Furthermore, with the new `amc init` command, containers can be created but not started.
+
+Starting containers with the `error` status is currently not possible, but will be allowed in the 1.18 release.
+
+### Dashboard improvements
+
+The Anbox Cloud Dashboard was extended in multiple ways. A new containers page allows management of containers present in AMS. This includes support for viewing log files of containers with the `error` status or opening a terminal for fast inspection.
+
+![Dashboard containers overview](images/dashboard-containers-overview.png)
+![Dashboard container terminal](images/dashboard-containers-terminal.png)
+
+To further provide information about a deployed cluster, the dashboard also provides an overview of the available LXD nodes and their properties.
+
+![Dashboard node overview](images/dashboard-nodes-overview.png)
+
+This will be further enhanced in future releases to allow easy modification of node settings and introduce support for performing maintenance tasks.
+
+### Other
+
+* Experimental support for software accelerated AV1 encoding is now available behind a [feature flag](https://anbox-cloud.io/docs/ref/ams-configuration) (`experimental.force_av1_software_encoding`).
+* Containers now connect to Anbox Stream Agent instead of Anbox Stream Gateway for WebRTC signaling.
+* HTTPS fingerprint checks were added to the Anbox runtime for deployment scenarios where the Anbox Stream Agent is not deployed with a certificate signed by a trusted CA.
+* Initial support for NUMA has been added to AMS. If a system has multiple NUMA nodes, AMS will select a node based on the locality of an additionally allocated GPU. If a container is not allocated with a GPU or if `cpu.limit_mode` is set to `pinning`, no specific NUMA is being selected.
+* To allow scheduling containers for applications onto specific nodes in a LXD cluster, an application in AMS can now add a `node-selector` part into it's manifest. This will tell the scheduler in AMS to schedule a container for the application only onto a node matching the criteria of the node selector. The node selector currently supports tag based selection only.
```

Comment:
```suggestion
* To allow scheduling containers for applications onto specific nodes in a LXD cluster, an application in AMS can now add a `node-selector` part into its manifest. This will tell the scheduler in AMS to schedule a container for the application onto a node only if it matches the criteria of the node selector. The node selector currently supports tag based selection only.
```